src/UserList/UserList.js

```diff
@@ -1,17 +1,29 @@
 import React from 'react';
 
 import { User } from '../User/User';
+import './userlist.scss';
 
 export const UserList = (props) => {
-  const result = props.list.map(item => (
-    <li key={item.id}>
-      <User name={item.name} surname={item.surname} age={item.age} />
-    </li>
-  ));
-
-  return (
-    <ul>
-      {result}
-    </ul>
-  );
+  let posts = '';
+  let result = '';
+  if (props.list) {
+    result = props.list.map(item => (
+      <li
+        key={item.name}
```

Comment:
dangerous to use _name_ field! But for this time ok)

Reply:
for some reason id was not unique - says debugger